UnitedIncome/components
# [APP-5108] Double Donut Component

### Description
> Please provide a brief description of your changes below.

Adds the double donut component. The API has been changed slightly during the port to make it more streamlined. It has all been documented in the docs file. 

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Mess around with the props, it should all make sense.
* You should be able to select each donut slice to expand the categories.
* Clicking the legend does not do anything. I'm unsure if this was the case before but that functionality doesn't seem present in production, and therefore it's not present in the port.
* You can now resize the double donut using the `height` and `width` props. There's an example in the story file.
* Click the very center of the donut 6 times for a surprise!

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

<img width="590" alt="Screen Shot 2020-05-11 at 4 30 34 PM" src="https://user-images.githubusercontent.com/10888441/81609081-3e294380-93a5-11ea-8805-1a65cd772b08.png">
<img width="322" alt="Screen Shot 2020-05-11 at 4 31 02 PM" src="https://user-images.githubusercontent.com/10888441/81609082-3e294380-93a5-11ea-8cfd-57659deba38b.png">

### Comments
> Any additional comments that will aid with the review process.

I'm doing my best to bring the test coverage up on this component. There's a _lot_ going on here but I don't want it to block any additional progress. 

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-5108)
